internal/cmd/cmd_sync_test.go

```diff
@@ -94,14 +94,14 @@ func (c *FakeClient) NewPublish(ctx context.Context) (gw.Publish, error) {
 	return &c.publishes[len(c.publishes)-1], nil
 }
 
-func (c *FakeClient) GetPublish(id string) gw.Publish {
+func (c *FakeClient) GetPublish(ctx context.Context, id string) (gw.Publish, error) {
 	for idx := range c.publishes {
 		if c.publishes[idx].id == id {
-			return &c.publishes[idx]
+			return &c.publishes[idx], nil
 		}
 	}
 	// Didn't find any, then return a broken one
-	return &BrokenPublish{id: id}
+	return &BrokenPublish{id: id}, fmt.Errorf("Broken publish due to no publish found")
```

Comment:
Relating to a comment I made on the real implementation:

I don't think we need BrokenPublish any more, this should just return `nil` if it fails.